docs/Templates/Templates.md

```diff
@@ -0,0 +1,271 @@
+    `<<[Percent]:”P”>>`  
+
+    - Percent (as Fixed-Point (`F`) Format Specifier):  
+    `<<[Percent]:”F”>>`  
+
+    - Percent (as Digit placeholder):  
+    `<<[Percent]:”#.##”>>`  
+
+    - Percent (as Percentage placeholder):  
+    `<<[Percent]:”%#0.00”>>` 
+
+
+- Date/time
+
+    - DateOfBirth (format `”dd.MM.yyyy”`):  
+    `<<[DateOfBirth]:”dd.MM.yyyy”>>`  
+
+    - DateOfBirth (format `”MMMM dd”`):  
+    `<<[DateOfBirth]:”MMMM dd”>>`  
+
+    - DateOfBirth (format `”MMMM dd” fr`):   
+    `<<[date.Format(DateOfBirth, ”MMMM dd”, “fr”)]>>`  
+
+    - DateOfBirth (format `”MMMM dd” en-GB`):  
+    `<<[date.Format(DateOfBirth, ”MMMM dd”, “en-GB”)]>>`  
+    
+    - Appointment (format `“dd.MMMM.yyyy HH:mm”`): 
+    `<<[Appointment]:”dd.MMMM.yyyy HH:mm“>>`  
+
+    - Appointment (format `“dddd, dd MMMM yyyy HH:mm:ss”`): 
+    `<<[Appointment]:”dddd, dd MMMM yyyy HH:mm:ss“>>`  
+
+    - Appointment (format `“dddd, dd MMMM yyyy HH:mm:ss” fr`): 
+    `<<[date.Format(Appointment, ”dddd, dd MMMM yyyy HH:mm:ss“, “fr”)]>>`  
+
+    - Appointment (format `“dddd, dd MMMM yyyy HH:mm:ss” en-GB`): 
+    `<<[date.Format(Appointment, ”dddd, dd MMMM yyyy HH:mm:ss“, “en-GB”)]>>`  
+
+#### Notes
+
+For the ASPOSE documentation about outputting expression results and their formatting please follow [this](https://docs.aspose.com/display/wordsnet/Template+Syntax#TemplateSyntax-OutputtingExpressionResults) guide.
+
+
+## Templates and document generation
+
+Logic Document Generation uses `.docx` templates and create documents out of them.  
+
+There are a few conventions that are followed when resolving a template from a configuration entry point of view (to know more about the configuration model visit the [Configuration](../Configuration/Configuration.md) documentation).  
+
+Logic Document Generation uses a hierarchical configuration model where each node is able to override a template from its parent node.  
+
+This design allows a master storage to define a set of generic templates for their customers to use, while allowing their customers to customize/extend all or some of them.   
+
+
+### How to override a Master template  
+
+#### Using SharePoint
+
+- Create a `.docx` file in your SharePoint Group.
+- Name it as the document you want to override and update its content as needed.  
+
+
+#### Using Azure Blob Storage
+
+- Upload a `.docx` file to your Azure Blob Storage.
+- Name it as the document you want to override and update its content as needed.  
+- Make sure you upload the file to the declared container and directory (`Blob prefix`) in Console for your configuration entry.  
+
+
+### How to hide a template from the list of available templates for a given entry
+
+#### Using SharePoint
+
+- Create an empty `.docx` file in your SharePoint Group.
+- Name it as the document you want to hide. 
+- For instance, if you want to hide `Template.docx`, the name of the document in your local SharePoint Group should be `Template.hidden.docx`
```

Comment:
@emiaj 
I might be wrong, I haven't tested the hidden thing, but I think the code says they should name it:

Template.docx.hidden

Reply:
If the code does say that, and we want it to be like what you've written, we should just raise a bug.

Reply:
Let me test this out first, I prefer to just adhere to whatever the convention is.

Reply:
Fixed in 47b4a50856a1cc1a6875c717a25c1a6efe596a76